scripts/health_check.sh

```
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+check_health() {
+    local target_port=$1
+    local target_env=$2
+
+    COUNTER=0
+    SUCCESS_COUNT=0
+    REQUIRED_SUCCESSES=2
+
+    while [ $SUCCESS_COUNT -lt $REQUIRED_SUCCESSES ] && [ $COUNTER -lt 12 ]; do
+        if curl --silent --fail http://localhost:$target_port/health; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)